lambdas/app.js

```diff
@@ -12,9 +12,8 @@ const originRequest = async (event, context) => {
     * Reads query string to check if S3 origin should be used, and
     * if true, sets S3 origin properties.
     */
-  const host = getHost(request);
-  const params = getSiteQueryParams(host, context.functionName);
-  return getSite(params)
+  const params = getSiteQueryParams(Utils.getHost(request), context.functionName);
+  return getSiteItem(params)
```

Comment:
we're already in `async` land here, might as well `await` `getSiteItem` for consistency.